pkg/minikube/bootstrapper/certs.go

```diff
@@ -197,3 +218,117 @@ func generateCerts(k8s config.KubernetesConfig) error {
 
 	return nil
 }
+
+func collectCACerts() (map[string]string, error) {
```

Comment:
Please add a comment here - describing where the certs are collected from.

Reply:
Sure.